src/addons/transitions/__tests__/ReactTransitionChildMapping-test.js

```diff
@@ -109,6 +109,7 @@ describe('ReactTransitionChildMapping', function() {
     });
   });
 
+  /*eslint-disable no-undef-init */
```

Comment:
We should either change this to `var next;` below or just turn off this rule everywhere if we don't care.